src/mysql.c

```diff
@@ -776,5 +777,6 @@ static int mysql_read (user_data_t *ud)
 
 void module_register (void)
 {
+	mysql_library_init(0, NULL, NULL);
```

Comment:
This happens to work but it'd be cleaner to do it in a plugin_init rather than in the registration.  For the short term purposes, it should be fine though.